sdk/appconfiguration/app-configuration/README.md

```diff
@@ -22,7 +20,7 @@ Use the App Configuration client library to:
 
 - Node.js version 8.x.x or higher
 
-### Install the Package
+### 1. Install the `@azure/app-configuration` package
```

Comment:
Nit: What's with the numbered list? I dont think that's in the template :)

Reply:
It's not - I added it in after watching one of the user studies where it seemed like they got lost navigating through the readme. So I thought the numbers would help make it more obvious that we're going through a series of steps (not choices, for instance).